db/migrate/20161019003515_create_linked_sites.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLinkedSites < ActiveRecord::Migration
+  def change
+    create_table :linked_sites do |t|
+      t.string :name
+      t.boolean :share_to
+      t.boolean :share_from
+      t.integer :link_type
```

Comment:
All of these should have `null: false` on them, since they're all switches.

Remember: a switch is either on or off.  You don't want a switch that can randomly cease existence